onboarding/source/controller.c

```diff
@@ -95,6 +113,9 @@ static void lightTimerCallback(TimerHandle_t xTimer) {
     /* USER CODE BEGIN */
     // Send light event to light service queue
 
+    ASSERT(xTimer != NULL);
+
+    sendToLightServiceQueue(MEASURE_LIGHT);
```

Comment:
`sendToLightServiceQueue` expects a pointer to an event of type `light_event_t`. There's an enum type defined in `amb_light_service.h` called `light_event_t`. This event would have a value of `MEASURE_LIGHT`.

So you'll need to do something like:
```
light_event_t event = MEASURE_LIGHT;
sendToLightServiceQueue(&event);
```